src/validation/nucleate_ice/CMakeLists.txt

```diff
@@ -25,14 +25,15 @@ endforeach()
 
 set(TEST_LIST
     nucleate_ice_cam_calc_merged # passing tolerance = 1.5e-5
-    hetero_merged
+    # hetero_merged
```

Comment:
are these tests just commented out for the moment, to be reintroduced in a subsequent PR?